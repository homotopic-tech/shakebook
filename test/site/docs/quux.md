---
title: The Quux In The Maw
description: bowowowowow
---

Lorem ipsum
