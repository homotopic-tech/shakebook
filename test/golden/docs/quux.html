<html>
  <head>
    <title>The Quux In The Maw</title>
  </head>
  <body>
    <h1>This post page is called The Quux In The Maw</h1>
    <p>It was generated from the source test/site/docs/quux.md</p>
    <p>For the url </p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the recent posts again:
      <ul>
      </ul>
    </p>
  </body>
</html>
